dts/bindings/rng/zephyr,psa-crypto-rng.yaml

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: PSA Crypto RNG/Entropy
+
+compatible: "zephyr,psa-crypto-rng"
+
+include: base.yaml
+
+properties:
+    label:
+      required: true
```

Comment:
We are no longer accepting required labels like this without strong justification; remove this part.